src/miscellaneous.css

```diff
@@ -14,6 +14,9 @@
  * <div class="scroll-styled scroll-auto h120"><div class='h240'></div></div>
  * <div class="scroll-styled scroll-styled--dark bg-gray-dark scroll-auto h120"><div class='h240'></div></div>
 */
+.scroll-styled {
```

Comment:
This doesn't strictly need to fall under `scroll-styled` right? Should this be added to `scroll-auto` instead?